modules/osd-aws-privatelink-architecture.adoc

```diff
@@ -14,7 +14,8 @@ The following diagram shows a multiple availability zone (Multi-AZ) PrivateLink
 
 image::156_OpenShift_ROSA_Arch_0621_privatelink.svg[Multi-AZ AWS PrivateLink cluster deployed on private subnets]
 
-= AWS reference architectures
+[id="osd-aws-reference-architecture.adoc_{context}"]
+== AWS reference architectures
```

Comment:
Same for this file - couldn't find this in a local dedicated preview, but this file's changes are easy to eyeball to say it looks fine.